nextflow.config

```
@@ -3,126 +3,152 @@
  *  nf-core/epitopeprediction Nextflow config file
  * -------------------------------------------------
  * Default config options for all environments.
- * Cluster-specific config options should be saved
- * in the conf folder and imported under a profile
- * name here.
  */
 
 // Global default params, used in configs
 params {
-
-  container = 'nfcore/epitopeprediction:latest' // Container slug. Stable releases should specify release tag!
-
-  help = false
+  // Pipeline specific options
   alleles = false
-  outdir = './results'
-
-  // input
-  somatic_mutations = false
-  peptides = false
-
   filter_self = false
-  wild_type = false
-  mhc_class = 'I'
+  mhc_class = 1
+  peptides = false
+  max_peptide_length = (mhc_class == 1) ? 11 : 16
+  min_peptide_length = (mhc_class == 1) ? 8 : 15
+  tools = 'syfpeithi'
   reference_genome = 'GRCh37'
```

```suggestion
  genome = 'GRCh37'
```
Could we have `genome` only instead?

---

for inspiration:
https://github.com/nf-core/sarek/blob/4bdff8eed3e4bf1a4393b226d2e87f09c2346d35/conf/test.config#L20-L22